includes/class-wp-job-manager-com-api.php

```diff
@@ -0,0 +1,172 @@
+<?php
+/**
+ * File containing the class \WP_Job_Manager\WP_Job_Manager_Com_API.
+ *
+ * @package wp-job-manager
+ * @since   $$next-version$$
+ */
+
+namespace WP_Job_Manager;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+/**
+ * \WP_Job_Manager\WP_Job_Manager_Com_API
+ */
+class WP_Job_Manager_Com_API {
+
+	const API_BASE_URL = 'https://wpjobmanager.com/wp-json/';
+
+	/**
+	 * The single instance of the class.
+	 *
+	 * @var self
+	 */
+	private static $instance = null;
+
+	/**
+	 * Get the main instance.
+	 *
+	 * @return self Main instance.
+	 */
+	public static function instance() {
+		if ( is_null( self::$instance ) ) {
+			self::$instance = new self();
+		}
+		return self::$instance;
+	}
+
+	/**
+	 * Get notices.
+	 *
+	 * @param int|null $max_age The max age (seconds) of the source data.
+	 *
+	 * @return array
+	 */
+	public function get_notices( $max_age = null ) {
+		$response = $this->request(
+			'wpjmcom-notices/1.0/notices',
```

Comment:
This changed while in review. This issue was hidden by the test snippet.
```suggestion
			'wpjmcom-notices/v1/notices',
```

Reply:
Good catch!